### ISSUE TITLE
Exits Server Mode on 1/0 metadata pages

### ISSUE THREAD

**Issue by Irodzuita** [9-FS/nhentai_archivist#32]
When running in server mode upon hitting a page that doesn't have any applicable results (easiest to replicate with the uploaded: < or > tag) it exits into client mode instead of abort iteration and sleep. I was hoping this was fixed in the last update with [d9815ae](https://github.com/9-FS/nhentai_archivist/commit/d9815aefcd6b30a1870fd48f350e13e73d425967) but was not. I'd like this to be added so that I can majorly reduce the amount of metadata pages I download for tags I only need the latest updates on. 

```
[2024-10-03T04:37:38] INFO  Connected to database at "./db/db.sqlite".
                      INFO  No hentai ID list found at "./config/downloadme.txt".
                      INFO  "NHENTAI_TAGS" are set.
[2024-10-03T04:37:39] WARN  Saving hentai metadata page 1 / <unknown> in database failed with: error returned from database: (code: 1) near ";": syntax error
                      INFO  Downloaded hentai metadata page 1 / 0.
                      INFO  Enter the holy numbers:
```

**Comment by 9-FS** [comment]
This is actually not a bug but was intended behaviour while I wrote the function. nHentai Archivist currently tries the following inputs in order:

1. downloadme.txt
1. tag search
1. manual input

And if the hentai ID list is still empty after trying out an input, it uses that as trigger to try the next one. This guarantees that the function always returns with some ID to process and made it easier to implement.

I can see though why this might be undesired. To clarify, is the following what you'd like to have implemented?

1. downloadme.txt
1. tag search OR manual input

**Comment by 9-FS** [comment]
This makes sense. Change is approved and I will come back to it as soon as I can.

**Comment by 9-FS** [comment]
published with version 3.5.0.
